data/is_ocaml_yet/gui.md

```diff
@@ -0,0 +1,77 @@
+---
+id: gui
+question: Is OCaml GUI Yet?
+answer: Not quite yet, but we're getting there!
+categories:
+  - name: Frameworks
+    status: 🟡
+    description: |
+      These are libraries written entirely in OCaml. It has packages for syndicating content and working with RSS feeds.
+    packages:
+      - name: bogue
+      - name: lablgtk
+      - name: lablqml
+      - name: tsdl
+      - name: sfml
+  - name: 3d graphics
```

Comment:
Do these names show up as titles or headers?

Reply:
![Display](https://github.com/ocaml/ocaml.org/assets/44207022/d015bd0c-e526-422f-9bec-292b959c44b5)
This is how the names are displayed.

Reply:
Thank you!